components/layout/blog-post/blog-post.hbs

```diff
@@ -0,0 +1,244 @@
+<div class="section--bg-prim-blue">
+
+    <div class="blog-post-hero section__inner">
+        <h1 class="blog-post-hero__title">If your team isn’t learning,
+            they’re falling behind. Fix it.</h1>
+        <div class="blog-post-hero__blog-author">
```

Comment:
Here and through elements below, having block elements prefixed by "blog" is a bit redundant because of the block it's in. Just use names like "blog-post-hero__author". A little shorter, less repetition.